docs/plugin-development.rst

```diff
@@ -804,6 +804,27 @@ This demonstrates one simple use case for client plugins, but using these same
 techniques, you should be able to do almost anything to change the core
 application as you need.
 
+JavaScript events
+*****************
+
+The JavaScript client handles events from both the server and client-specific
+code.  The server events originate on the server and can be monitored by both
```

Comment:
Thanks.  I had realized this was inaccurate and could be clearer, but then forgot to rework it.